step-templates/import-databricks-workbooks.json

```
@@ -0,0 +1,76 @@
+  },
+  "Parameters": [
+    {
+      "Id": "b8cd5d73-29d9-4ba2-bdfa-86cc1316a16f",
+      "Name": "DeployDataBricksWorkBookPackage",
+      "Label": "DataBricks WorkBook Package",
+      "HelpText": "The Databricks workbook package",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Package"
+      }
+    },
+    {
+      "Id": "f81b9ccb-beea-4d8f-a049-ee9ea6da643e",
+      "Name": "DataBricksInstanceUri",
+      "Label": "Databricks Instance Uri",
+      "HelpText": "The Databricks Instance URL",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "6c1aed25-f9a6-4c1f-b8e9-d50cc8669f2d",
+      "Name": "AccessToken",
```

This parameter name should be prefixed as the other parameters are e.g. `DataBricksAccessToken` (you'll also need to update any usages internally in your script)
```suggestion
      "Name": "DataBricksAccessToken",
```